apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -94,6 +94,7 @@ export class VersionAction extends BaseRushAction {
   }
 
   protected run(): Promise<void> {
+    this.runRushCheckIfNecessary();
```

Comment:
>this.runRushCheckIfNecessary(); [](start = 4, length = 31)

This should happen at the end, to confirm that `rush version` didn't introduce a problem.

Reply:
I think it should happen at start and end.

---
In reply to: [218978422](https://github.com/Microsoft/web-build-tools/pull/829#discussion_r218978422) [](ancestors = 218978422)